config.py

```diff
@@ -88,9 +88,12 @@ class Config(object):
     @staticmethod
     def init_app(app):
         repo_root = os.path.abspath(os.path.dirname(__file__))
+        digitalmarketplace_govuk_frontend = os.path.join(repo_root, "node_modules", "digitalmarketplace-govuk-frontend")
         template_folders = [
             os.path.join(repo_root, "app", "templates"),
-            os.path.join(repo_root, "node_modules", "digitalmarketplace-govuk-frontend", "govuk-frontend"),
+            os.path.join(digitalmarketplace_govuk_frontend),
+            os.path.join(digitalmarketplace_govuk_frontend, "digitalmarketplace", "templates"),
+            os.path.join(digitalmarketplace_govuk_frontend, "govuk-frontend"),
```

Comment:
It's this line that is blocking the error templates from loading - it works fine on the Admin FE though? 🤔